x/group/keeper/keeper.go

```diff
@@ -73,16 +73,26 @@ type Keeper struct {
 	voteByVoterIndex    orm.Index
 
 	router *authmiddleware.MsgServiceRouter
+
+	maxMetadataLength int // MaxMetadataLength defines the max length of the metadata bytes field for various entities within the group module
```

Comment:
Maybe this comment can go to the KeeperParams struct, since that once is public?

```suggestion
	maxMetadataLength int // MaxMetadataLength defines the max length of the metadata bytes field for various entities within the group module. Defaults to 255 if not explicitly set.
```